Common/src/Pollster/TaskHandler.cs

```diff
@@ -496,14 +509,14 @@ await agentHandler_.Stop(CancellationToken.None)
       if (reply.Output.TypeCase is Output.TypeOneofCase.Ok)
       {
         logger_.LogDebug("Complete processing of the request");
-        await agent_.FinalizeTaskCreation(CancellationToken.None)
+        await agent_.FinalizeTaskCreation(postCancellationToken)
                     .ConfigureAwait(false);
       }
 
       await submitter_.CompleteTaskAsync(taskData_,
                                          false,
                                          reply.Output,
-                                         CancellationToken.None)
+                                         postCancellationToken)
```

Comment:
In these last calls, I'd rather have no cancellationtoken : these are really the very last thing that are being done for the task. If there is a chance, even small that it can be done before the container is killed, we should take it.